app/models/YourDetails.scala

```diff
@@ -36,24 +36,25 @@ object YourDetails {
       "firstName" -> optional(text)
         .verifying("error.firstNameRequired", _.isDefined)
         .verifying("error.firstNameLength", i => i.isEmpty || i.getOrElse("").length <= 35)
-        .verifying("error.firstNamePattern", i => i.isEmpty || i.getOrElse("").length > 35 || i.getOrElse("").matches("""^[A-Za-z0-9 \-,.&'\\]{1,35}$""")),
+        .verifying("error.firstNamePattern", i => i.isEmpty || i.getOrElse("").length > 35 || i.getOrElse("").matches("""^[A-Za-z \-']{1,35}$""")),
 
       "lastName" -> optional(text)
         .verifying("error.lastNameRequired", _.isDefined)
         .verifying("error.lastNameLength", i => i.isEmpty || i.getOrElse("").length <= 35)
-        .verifying("error.lastNamePattern", i => i.isEmpty || i.getOrElse("").length > 35 || i.getOrElse("").matches("""^[A-Za-z0-9 \-,.&'\\]{1,35}$""")),
+        .verifying("error.lastNamePattern", i => i.isEmpty || i.getOrElse("").length > 35 || i.getOrElse("").matches("""^[A-Za-z \-']{1,35}$""")),
 
       "role" -> optional(text)
         .verifying("error.roleRequired", _.isDefined)
         .verifying("error.roleLength", i => i.isEmpty || i.getOrElse("").length <= 30)
-        .verifying("error.rolePattern", i => i.isEmpty || i.getOrElse("").length > 30 || i.getOrElse("").matches("""^[A-Za-z0-9 \-,.&'\/]{1,30}$""")),
+        .verifying("error.rolePattern", i => i.isEmpty || i.getOrElse("").length > 30 || i.getOrElse("").matches("""^[A-Za-z \-']{1,30}$""")),
 
       "phone" -> optional(text)
         .verifying("error.phoneRequired", _.isDefined)
         .verifying("error.phonePattern", i => i.isEmpty || i.getOrElse("").matches("""^[A-Z0-9 \)\/\(\*\#\-\+]{1,24}$""")),
 
-      "email" -> optional(email)
+      "email" -> optional(text)
         .verifying("error.emailRequired", _.isDefined)
+        .verifying("error.email", i => i.isEmpty || i.getOrElse("").matches("""^[a-zA-Z0-9_+&*-](?:\.[a-zA-Z0-9_+&*-])*@(?:[a-zA-Z0-9-].)[a-zA-Z]{2,7}$""")),
```

Comment:
I know this regex was in the ticket but i put a couple of emails in and all fail validation, also causing the ATs to fail.

Reply:
Hmm yeah, this regex doesn't seem to work at all...
Just testing an updated one now